test/simple_select.test.js

```diff
@@ -0,0 +1,348 @@
+/* eslint no-shadow:[0] */
+import test from 'tape';
+import GLDraw from '../';
+import { createMap, cloneFeature } from './test_utils';
+import spy from 'sinon/lib/sinon/spy'; // avoid babel-register-related error by importing only spy
+import stub from 'sinon/lib/sinon/stub'; // avoid babel-register-related error by importing only stub
+import AfterNextRender from './utils/after_next_render';
+import makeMouseEvent from './utils/make_mouse_event';
+
+test('simple_select', t => {
+
+  var map = createMap();
+  spy(map, 'fire');
+
+  var Draw = GLDraw();
+  map.addControl(Draw);
+
+  var afterNextRender = AfterNextRender(map);
+
+  var cleanUp = function(cb) {
+    Draw.deleteAll();
+    map.fire.reset();
+    afterNextRender(cb);
+  }
+
+  var getFireArgs = function() {
+    var args = [];
+    for (var i=0; i<map.fire.callCount; i++) {
+      args.push(map.fire.getCall(i).args);
+    }
+    return args;
+  }
+
+  t.test('simple_select - setup', t => {
+    var donedone = false;
+    var done = function() {
+      if (donedone === false) {
+        t.pass('map loaded');
+        t.end();
+        donedone = true;
+      }
+    };
+    if (map.loaded()) {
+      done();
+    }
+    map.on('load', done);
+  });
+
+
+  t.test('simple_select box select', t => {
+    Draw.add(cloneFeature('negitivePoint'));
+    var id = Draw.add(cloneFeature('point'));
```

Comment:
I think it would make the test more transparent to specify the coordinates within this function. Right now we're trusting that the point you're cloning is within the box you specify below; but it could be made explicit.